src/advanced_inv.cpp

```diff
@@ -1432,6 +1436,10 @@ void advanced_inventory::display()
         ui->mark_resize();
 
         ui->on_redraw( [&]( const ui_adaptor & ) {
+            if( always_recalc ) {
+                recalc = true;
+            }
```

Comment:
```suggestion
            recalc |= always_recalc;
```
I'd write it like this, but it's up to you.